src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/matmul_weights_decompression.cpp

```diff
@@ -130,107 +91,6 @@ class MatmulWeightsDecompression : public testing::WithParamInterface<MatmulWeig
     }
 
 protected:
-    std::shared_ptr<ov::Node> initDecompressionWeights(const ov::Shape& weights_shape,
-                                                       const int group_size,
-                                                       const ov::element::Type data_precision,
-                                                       const ov::element::Type weights_precision,
-                                                       const ov::element::Type decompression_precision,
-                                                       const bool transpose_weights,
-                                                       const DecompressionSubtractType decompression_subtract_type,
-                                                       const bool reshape_on_decompression_constant) {
```

Comment:
This builder is moved to weights_decompression_builders.hpp and renamed to `initMatMulDecompressionSubgraph`